apps/storybook/stories/components/alert/Alert.mdx

```diff
@@ -1,41 +1,62 @@
 import { Title, Canvas, Meta, Story, Controls } from "@storybook/blocks";
 import * as AlertStories from "./Alert.stories";
+import { DocsHeading, DocsAnatomy, DocsStoriesContainer, DocsStory } from "../../templates";
+import { Box } from "@kvib/react/src";
 
 <Meta of={AlertStories} />
 
 # Alert
 
-Se [https://chakra-ui.com/docs/components/alert](https://chakra-ui.com/docs/components/alert) for eksempler og dokumentasjon.
-
-## Ta i bruk
+Alert brukes til meldinger som kommuniserer informasjon uten at brukeren trenger å gjøre noe.
 
 ```jsx
 import { Alert, AlertIcon, AlertTitle, AlertDescription } from "@kvib/react";
 ```
 
-## Props
+<DocsHeading>Props</DocsHeading>
 
 <Canvas of={AlertStories.Alert} />
 <Controls of={AlertStories.Alert} />
 
-## Statuser
-
-<Canvas of={AlertStories.AlertStatus} />
-
-## Farger
-
-<Canvas of={AlertStories.AlertColors} />
-
-## Varianter
-
-<Canvas of={AlertStories.AlertVariants} />
-
-## Egenkomponert alert
-
-<Canvas of={AlertStories.AlertComposition} />
-
-## Kjente feil og mangler
-
-Bruk av colorScheme="orange" og variant="solid" i sammen gir for lav kontrast. Dette fargetemaet er også brukt med status="warning", som dermed heller ikke kan brukes sammen med variant="solid".
+<DocsHeading>Anatomy</DocsHeading>
```

Comment:
Lurer på om jeg kanskje ville ha flyttet anatomy over props, og samle props og options, siden dette er to ting som egentlig beskriver det samme.

Reply:
Jeg ser på NAV at de har egnet til, og uegnet til helt øverst på siden, rett under den korte beskrivelsen av komponenten. Det liker jeg! Og så hadde det vært fint med lenke til bedre alternativer sammen med uegnet til-punktene.